src/SockPerf.cpp

```diff
@@ -312,6 +312,10 @@ static const AOPT_DESC  client_opt_desc[] =
 		OPT_OUTPUT_PRECISION, AOPT_NOARG,	aopt_set_literal( 0 ),	aopt_set_string( "increase_output_precision" ),
 		"Increase number of digits after decimal point of the throughput output (from 3 to 9). "
 	},
+	{
+		OPT_DUMMY_SEND, AOPT_OPTARG,	aopt_set_literal( 0 ),	aopt_set_string( "dummy-send" ),
+		"Use dummy-send instead of busy wait. optional: set dummy-send rate per second (default 10000). usage: --dummy-send <rate>"
```

Comment:
fix:
1. 10,000 (use comma)
2. option: "[<rate>]"
3. add "MAX" as option
4. explain that rate value "must be lower than --mps"